### PR TITLE
Elasicsearch V2 -  added test of the fetch-incident related parameters to the test-module

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [XSUP-44925](https://jira-dc.paloaltonetworks.com/browse/XSUP-44925)

## Description
- As shown in the [XSUP-44925](https://jira-dc.paloaltonetworks.com/browse/XSUP-44925) bug, the current `test-module` doesn't check fetch-incidents related parameters. Therefore, I added those checks to the test module. 
- Removed the fetch incidents required parameters from the 'Advanced' display (in the advanced display they are first hidden, which make no sence for required parameters). 
- Added explanation about the limitation of the Test module in this integration (relates to this ticket - [XSUP-32388](https://jira-dc.paloaltonetworks.com/browse/XSUP-32388)) and a recommendation to use the `!es-integration-health-check` command to test the fetch incidents configuration.  

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 1]
```diff
@@ -50,21 +50,28 @@ configuration:
 - display: Query String
   name: fetch_query
   type: 0
-  additionalinfo: The query will be used when fetching incidents. Index time field will be used as a filter in the query.
+  additionalinfo: |- 
+   The query will be used when fetching incidents. Index time field will be used as a filter in the query.
+
+   The integration test button doesn't fully tests the fetch incidents validity. To verify that you instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```
Comment (yuvalbenshalom) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:56]:
```suggestion
   The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 2]
```diff
@@ -50,21 +50,28 @@ configuration:
...
   name: fetch_time_field
   type: 0
   section: Collect
-  advanced: true
   required: false
-  additionalinfo: The time field on which sorting and limiting are performed. If using a nested field, separate field names using dot notation.
+  additionalinfo: |-
+   The time field on which sorting and limiting are performed. If using a nested field, separate field names using dot notation.
+
+   The integration test button doesn't fully tests the fetch incidents validity. To verify that you instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```
Comment (yuvalbenshalom) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:67]:
```suggestion
   The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_description.md [hunk 3]
```diff
@@ -51,4 +51,7 @@ The types of time-fields supported are:
    - **Timestamp-Second** - A number referring to seconds since epoch (midnight, 1 January 1970). For example: '1572164838'.
    - **Timestamp-Milliseconds** - A number referring to milliseconds since epoch (midnight, 1 January 1970). For example: '1572164838123'.
 
-Note: Not all fields can be sorted in Elasticsearch. The fields are used to sort the results table.  The supported result types are boolean, numeric, date, and keyword fields.
+Notes:
+- Not all fields can be sorted in Elasticsearch. The fields are used to sort the results table.
+  The supported result types are boolean, numeric, date, and keyword fields.
+- The integration test button doesn't fully tests the fetch incidents validity. To verify that you instance is set up correctly for fetching incidents please run the ***!es-integration-health-check*** command.
```
Comment (yuvalbenshalom) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_description.md:57]:
```suggestion
- The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the ***!es-integration-health-check*** command.
```

**Review by yuvalbenshalom — APPROVED** [review]
See my comments

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 4]
```diff
@@ -50,21 +50,28 @@ configuration:
 - display: Query String
   name: fetch_query
   type: 0
-  additionalinfo: The query will be used when fetching incidents. Index time field will be used as a filter in the query.
+  additionalinfo: |- 
+   The query will be used when fetching incidents. Index time field will be used as a filter in the query.
+
+   The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:56]:
```suggestion
   The integration test button doesn't fully test the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents, run the '!es-integration-health-check' command.
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 5]
```diff
@@ -50,21 +50,28 @@ configuration:
...
   name: fetch_time_field
   type: 0
   section: Collect
-  advanced: true
   required: false
-  additionalinfo: The time field on which sorting and limiting are performed. If using a nested field, separate field names using dot notation.
+  additionalinfo: |-
+   The time field on which sorting and limiting are performed. If using a nested field, separate field names using dot notation.
+
+   The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:67]:
```suggestion
   The integration test button doesn't fully test the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents, run the '!es-integration-health-check' command.
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 6]
```diff
@@ -50,21 +50,28 @@ configuration:
...
 - display: Raw Query
   name: raw_query
   type: 12
-  additionalinfo: Will override the 'Query String' Lucene syntax string. Results will not be filtered.
+  additionalinfo: |- 
+   Will override the 'Query String' Lucene syntax string. Results will not be filtered.
+
+   The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the '!es-integration-health-check' command.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:74]:
```suggestion
   The integration test button doesn't fully test the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents, run the '!es-integration-health-check' command.
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_description.md [hunk 7]
```diff
@@ -51,4 +51,7 @@ The types of time-fields supported are:
    - **Timestamp-Second** - A number referring to seconds since epoch (midnight, 1 January 1970). For example: '1572164838'.
    - **Timestamp-Milliseconds** - A number referring to milliseconds since epoch (midnight, 1 January 1970). For example: '1572164838123'.
 
-Note: Not all fields can be sorted in Elasticsearch. The fields are used to sort the results table.  The supported result types are boolean, numeric, date, and keyword fields.
+Notes:
+- Not all fields can be sorted in Elasticsearch. The fields are used to sort the results table.
+  The supported result types are boolean, numeric, date, and keyword fields.
+- The integration test button doesn't fully tests the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents please run the ***!es-integration-health-check*** command.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_description.md:57]:
```suggestion
- The integration test button doesn't fully test the fetch incidents validity. To verify that the instance is set up correctly for fetching incidents, run the ***!es-integration-health-check*** command.
```

File: Packs/Elasticsearch/ReleaseNotes/1_3_25.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Elasticsearch v2
+
+- Fixed an issue where the fetch-incidents-related parameters were not tested by clicking the integration test button.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/ReleaseNotes/1_3_25.md:6]:
```suggestion
Fixed an issue where the fetch-incidents-related parameters were not tested by clicking the integration test button.
```

**Comment by ShirleyDenkberg** [comment]
@yuvalbenshalom Doc review completed.

**Pull request merged**